TWLight/users/forms.py

```diff
@@ -133,18 +133,21 @@ class Meta:
         model = UserProfile
         fields = ["terms_of_use"]
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, user_profile, *args, **kwargs):
         super(TermsForm, self).__init__(*args, **kwargs)
 
         # Translators: Users must click this button when registering to agree to the website terms of use.
         self.fields["terms_of_use"].label = _("I agree with the terms of use")
-
+        self.fields["terms_of_use"].initial = user_profile.terms_of_use
         self.helper = FormHelper()
         self.helper.form_class = "form-inline"
         self.helper.field_template = "bootstrap3/layout/inline_field.html"
-
+        self.fields["terms_of_use"].help_text = _("By unchecking this box and clicking “Update”, " \
+                                                 "you may explore the site, but you will not be able to apply for " \
+                                                 "access to materials or evaluate applications unless you agree " \
+                                                 "with the terms of use.")
```

Comment:
This translatable string needs a translator comment. Additionally, when you run Black (our code formatter), this will get reformatted as follows:
```
        self.fields["terms_of_use"].help_text = _(
            "By unchecking this box and clicking “Update”, "
            "you may explore the site, but you will not be able to apply for "
            "access to materials or evaluate applications unless you agree "
            "with the terms of use."
        )
```
But, in order for Django to pick up the translator comment immediately above, the string should start in the same line as `_(`. So, you might want to turn of Black for this piece of code `# fmt: off ... # fmt: on` ([example](https://github.com/WikipediaLibrary/TWLight/pull/455/files#diff-4a27f05041fdff3f35fb521e35368043R410-R413)) and move the closing `)` to the same line as `_(` (i.e. the entire string in one line).

Reply:
Hello Aaron,  
Thank you for the update.  I followed the example you shared.  Please let me know if you have any questions.
Kangyoon